### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 11

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#311] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152322386-e46a97ad-ba64-4eb8-a399-cca3841b1b79.png)

**Comment by Exyss** [comment]
![Esercizio 11](https://user-images.githubusercontent.com/65043207/152647572-dde8af9e-dcac-4286-a7f1-5039a93ce4fc.jpg)

**Comment by Elia-Belli** [comment]
Nella soluzione di @Exyss alla seconda riga del Passo Induttivo PI c'è un errore, nel portare fuori il termine n+1esimo dalla sommatoria ha dimenticato di elevare il denominatore alla seconda: $\frac{1}{n+1}$ anzichè $\frac{1}{(n+1)^2}$.

**Dimostrazione per Induzione**

- **Caso Base**: $n=2$ , $1+1/4 < 2-1/2 \rightarrow 5/4<3/2$;
- **Passo induttivo**: $n \geq 2$ , $P(n) \Rightarrow P(n+1)$:
  $P(n+1) = \sum\limits_{i=1}^{n+1}{\frac{1}{i^2}} < 2 - \frac{1}{n+1} \rightarrow \sum\limits_{i=1}^{n}{\frac{1}{i^2}} + \frac{1}{(n+1)^2}< 2 - \frac{1}{n+1}$, applichiamo l'ipotesi induttiva $\sum\limits_{i=1}^{n}{\frac{1}{i^2}} < 2 - \frac{1}{n}$
  otteniamo $\sum\limits_{i=1}^{n+1}{\frac{1}{i^2}} +\frac{1}{(n+1)^2}< 2 - \frac{1}{n} + \frac{1}{(n+1)^2}$ , quindi ci basta mostrare che $2 - \frac{1}{n} + \frac{1}{(n+1)^2} < 2 - \frac{1}{n+1}$.
  $-\frac{1}{n} + \frac{1}{n+1} + \frac{1}{(n+1)^2} < 0$ , facendo l'mcm e sviluppando i termini si ottiene $\frac{-1}{n(n+1)^2} <0$ che è ovviamente vera. $Q.E.D.$
